pkg/components/estimator/estimator.go

```diff
@@ -28,7 +28,7 @@ import (
 
 const (
 	// NOTE: update tests/images.yaml when changing this image
-	StableImage          = "quay.io/sustainable_computing_io/kepler_model_server:v0.7.11"
+	StableImage          = "quay.io/sustainable_computing_io/kepler_model_server:v0.7.11-2"
```

Comment:
@vprashar2929 we should also change the entrypoint and command
Please check the model-server compose files.

Reply:
Yeah forgot about that 😅